recipes/syft/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "syft" %}
+{% set version = "0.2.2a2" %}
+
+package:
+  name: syft
+  version: "0.2.2a2"
+
+source:
+  url: https://files.pythonhosted.org/packages/97/ba/c01bb914f395611034bab10cb8d1d22e49691534d30f85efbd82f7bd27cf/syft-0.2.0a2.tar.gz
+  sha256: f3839e5be537c3d9b3f530778e9f08cdeeab99d9ba71f85c35b158a2264c7fc6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+  # work on later
+    - simplejson
+    - simplejson.tests
```

Comment:
What should I replace tests with exactly

Reply:
@npavlovikj currently the build breaks at testing. I need to replace the default tests template with pysyft ones. But not sure exactly with what.